src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.cs

```diff
@@ -50,8 +50,9 @@ private enum TimeZoneInfoResult
         // constants for TimeZoneInfo.Local and TimeZoneInfo.Utc
         private const string UtcId = "UTC";
         private const string LocalId = "Local";
+        private const string InvariantUtcStandardDisplayName = "Coordinated Universal Time";
 
-        private static readonly TimeZoneInfo s_utcTimeZone = CreateCustomTimeZone(UtcId, TimeSpan.Zero, "(UTC) Coordinated Universal Time", "Coordinated Universal Time");
```

Comment:
>"(UTC) Coordinated Universal Time", "Coordinated Universal Time") [](start = 104, length = 65)

I recall we had a problem before if the names in utc is changed. I don't recall exactly what was the problem but what I recall is some other created utc was not equal to this one because of the name. maybe would be safer to keep this hardcoded names here.

Reply:
I think it's important to not hardcode things to English unless we're specifically talking about the Invariant culture or fallbacks.  It is a bit strange when your current culture is something like say, `ja-JP`, and thus all your other time zones are in Japanes using Kanji characters but UTC is in English...  

That said, since we put all the display names as properties directly on `TimeZoneInfo`, the only culture that can be used presently is the `CurrentUICulture`, on Unix anyway (as you know on Windows it uses the OS locale only due to data coming from language packs).  Thus, it seems like to change locales, one would have to set the `CurrentUICulture` and then call `TimeZoneInfo.ClearCachedData`.  That will work with all time zones *except* UTC with my current PR.  I think I should change it such that UTC is part of the cached data and not a static.

There really shouldn't be any equality comparison based on the localized strings either way.  We either do reference equality to the UTC `TimeZoneInfo` object, or a regular comparison using the `Id` property.

Reply:
I am not objecting what you are saying. I am just telling we ran into some issue before when this is changed before.

Reply:
I believe the problem may have been due to how [BinaryFormatterTestData.cs](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs) has a blob for `TimeZoneInfo.Utc` with "Coordinated Universal Time" embedded in it.

I added an exclusion to `BinaryFormatterTests.SanityCheckBlob` to only test `TimeZoneInfo` objects on English platforms.